lib/super_digit.rb

```diff
@@ -1,15 +1,23 @@
 # Superdigit
 
-# Time Complexity - ?
-# Space Complexity - ?
+# Time Complexity - O(n^2)
+# Space Complexity - O(n^2)
 def super_digit(n)
```

Comment:
This works, but your time complexity is off.  The number of times this repeats depends on the number of digits. So O(log<sub>10</sub>n)